### PR TITLE
Fix for Tree view is not refreshed for PDS member actions (issue #398)

### PR DESCRIPTION
**Pull request by katelynienaber**
Fix for issue #398...this time without credentials ^^''

Signed-off-by: katelynienaber <katelyn.nienaber@broadcom.com>

### CODE REVIEW

**Comment by zFernand0** [comment]
It works for me :)

One thing I noticed is that it doesn't refresh the matching favorite.
For example if I have a pds in my favorites and I searched for it (i.e. it's in my current view), when I upload a member on via the favorite, the view doesn't update.

Also, the top level refresh does not close PDSs on favorites, only on the view.

The issue in question is resolved though 👍

**Comment by jellypuno** [comment]
> The issue in question is resolved though

That's the most important thing 😄 👍

**Comment by katelynienaber** [comment]
> It works for me :)
> 
> One thing I noticed is that it doesn't refresh the matching favorite.
> For example if I have a pds in my favorites and I searched for it (i.e. it's in my current view), when I upload a member on via the favorite, the view doesn't update.
> 
> Also, the top level refresh does not close PDSs on favorites, only on the view.
> 
> The issue in question is resolved though 👍

Hello again :)

I added a bit more to extension.js so now favorites should be refreshed too.

Also some additions to the unit tests for these issues.

**Comment by zFernand0** [comment]
No need to implement the top-level refresh all right away. 
We can open an issue and put it in the backlog (if you have something else you'd like to focus on)

**Comment by katelynienaber** [comment]
> No need to implement the top-level refresh-all right away.
> We can open an issue and put it in the backlog (if you have something else you'd like to focus on)

Okay. I am nor sure I got everything but I will try not to worry about it :)

Anyway Christmas is coming so I probably won't start anything too big until January... ❄️

**Comment by Colin-Stone** [comment]
> Also, it'd be nice if we could capture this in a test :)

Were you able to create a new test as Frenando requested?.

**Comment by katelynienaber** [comment]
> > Also, it'd be nice if we could capture this in a test :)
> 
> Were you able to create a new test as Frenando requested?.

Yes, it's been added in `mvsNodeActions.unit.test.ts` and `extension.unit.test.ts`. I checked to see if refresh was called with the correct node.

**Comment by Colin-Stone** [comment]
@katelynienaber Why has this PR been closed?. I hasn't actually been merged into master?

**Comment by katelynienaber** [comment]
> @katelynienaber Why has this PR been closed?. It hasn't actually been merged into master has it?.
> It appears as if there is just one test file changed but I was sure it had multiple changes earlier.

I didn't mean to close it. Also I had to push again from my computer, I think I pushed to my repo's master branch at some point with the wrong version. This one has the most-recently tested code, I checked the refresh functionality and the unit tests last night before I updated the code here...

**Comment by Colin-Stone** [comment]
I couldn't work out what had happened and my main concern was that instead of all of the files and  changes I had seen previously whilst looking at the PR we have just this one test file now. It was as if we had lost or overwritten your other changes so I haven't included this PR in the 1.1 release. Hopefully you can work out if the changes are okay or anything is missing.

**Comment by katelynienaber** [comment]
> I couldn't work out what had happened and my main concern was that instead of all of the files and changes I had seen previously whilst looking at the PR we have just this one test file now. It was as if we had lost or overwritten your other changes so I haven't included this PR in the 1.1 release. Hopefully you can work out if the changes are okay or anything is missing.

Yes I will look again tonight before I leave.

The changes were indeed overwritten on GitHub at some point, so I added them back to the code I have on my computer last night, and debugged the refresh functions/ran unit tests one more time.

The code I have on my PC is correct, I will just push again tonight to make sure it is the same here.

**Comment by katelynienaber** [comment]
Hello again, so I checked the code on my master branch (katelynienaber/bscode-extension-for-zowe) and it shows the changes there.

In these locations:
779 - 818 (extension.ts)
1213, 1218, 1219 (extension.unit.test.ts)

File: i18n/sample/src/Profiles.i18n.json [hunk 1]
```diff
@@ -18,7 +18,7 @@
     "promptcredentials.option.prompt.username.placeholder": "User Name",
     "promptcredentials.option.prompt.username": "Enter the user name for the connection",
     "promptcredentials.invalidusername": "Please enter your z/OS username. Operation Cancelled",
-    "promptcredentials.option.prompt.password.placeholder": "Password",
+    "promptcredentials.option.prompt.passWord.placeholder": "Password",
```
Comment (zFernand0) [inline i18n/sample/src/Profiles.i18n.json]: I believe it was decided to go with lowercase `password`. not sure how this got in here.

**Comment by zFernand0** [comment]
I recommend just reverting back to https://github.com/zowe/vscode-extension-for-zowe/pull/404/commits/02b4656020a36fdc8967a115f4206b4e58d38123

**Comment by zFernand0** [comment]
#404 - Functionality not found 😋

**Comment by zFernand0** [comment]
Closing in favor of https://github.com/zowe/vscode-extension-for-zowe/pull/464#issuecomment-577577027

**Pull request closed without merging**